modules/common/proto/header.proto

```diff
@@ -32,4 +32,7 @@ message Header {
   optional StatusPb status = 8;
 
   optional string frame_id = 9;
+
+  // The processing time in ms
+  optional double processing_time_ms = 10;
```

Comment:
How about in seconds? As the Clock::Now() returns seconds.

Reply:
We can use std::chrono::system_clock::now(), instead